<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Benutzerübersicht</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget">
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <item>
         <widget class="QPushButton" name="btnToggleView">
          <property name="text">
           <string>Ansicht umschalten</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeUsers">
          <property name="headerLabels" stdset="0">
           <stringlist>
            <string>Benutzer</string>
           </stringlist>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget">
       <layout class="QVBoxLayout" name="verticalLayoutRight">
        <item>
         <widget class="QTableWidget" name="tableEntries">
          <property name="columnCount">
           <number>5</number>
          </property>
          <property name="horizontalHeaderLabels" stdset="0">
           <stringlist>
            <string>Artikelnummer</string>
            <string>Beschreibung</string>
            <string>Größe</string>
            <string>Preis</string>
            <string>Datum</string>
           </stringlist>
          </property>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
